server/src/entities/Character.ts

```diff
@@ -0,0 +1,21 @@
+import { Entity, PrimaryKey, Property } from "mikro-orm";
+import { Field, ObjectType } from "type-graphql";
+
+@ObjectType()
+@Entity()
+export class Character {
+
+  @Field()
+  @PrimaryKey()
+  id!: number;
+
+  @Field(() => String)
+  @Property({type: 'text', unique: true})
+  username!: string;
```

Comment:
This should be the display name not the username right?

Reply:
now all the name is used as username, so that it wont be messy.